web/src/admin/roles/RoleViewPage.ts

```diff
@@ -83,73 +81,61 @@ export class RoleViewPage extends AKElement {
-                    slot="page-overview"
-                    data-tab-title="${msg("Overview")}"
-                    class="pf-c-page__main-section pf-m-no-padding-mobile"
-                >
-                    <div class="pf-l-grid pf-m-gutter">
-                        <div
-                            class="pf-c-card pf-l-grid__item pf-m-12-col pf-m-3-col-on-xl pf-m-3-col-on-2xl"
-                        >
-                            <div class="pf-c-card__title">${msg("Role Info")}</div>
-                            <div class="pf-c-card__body">
-                                <dl class="pf-c-description-list">
-                                    <div class="pf-c-description-list__group">
-                                        <dt class="pf-c-description-list__term">
-                                            <span class="pf-c-description-list__text"
-                                                >${msg("Name")}</span
-                                            >
-                                        </dt>
-                                        <dd class="pf-c-description-list__description">
-                                            <div class="pf-c-description-list__text">
-                                                ${this._role.name}
-                                            </div>
-                                        </dd>
-                                    </div>
-                                </dl>
-                            </div>
-                        </div>
-                        <div
-                            class="pf-c-card pf-l-grid__item pf-m-12-col pf-m-9-col-on-xl pf-m-9-col-on-2xl"
-                        >
-                            <div class="pf-c-card__title">
-                                ${msg("Assigned global permissions")}
-                            </div>
-                            <div class="pf-c-card__body">
-                                <ak-role-permissions-global-table
-                                    roleUuid=${this._role.pk}
-                                ></ak-role-permissions-global-table>
-                            </div>
+
+        return html` <ak-tabs>
+            <section
+                slot="page-overview"
+                data-tab-title="${msg("Overview")}"
+                class="pf-c-page__main-section pf-m-no-padding-mobile"
+            >
+                <div class="pf-l-grid pf-m-gutter">
+                    <div
+                        class="pf-c-card pf-l-grid__item pf-m-12-col pf-m-3-col-on-xl pf-m-3-col-on-2xl"
+                    >
+                        <div class="pf-c-card__title">${msg("Role Info")}</div>
+                        <div class="pf-c-card__body">
+                            ${renderDescriptionList([
+                                [msg("Name"), this._role.name],
+                                [msg("Edit"), this.renderUpdateControl(this._role)],
+                            ])}
```

Comment:
Nice, small description list renderer.  :-)

Reply:
can we turn this into a component? Currently when using this we'll also have to manually import the descriptor CSS

Reply:
We can, but you already have in existing cases, so I went with a lazier solution.  I do intend to create a `ak-description-list` component someday.  This is a compromise; I think it's a good one that doesn't introduce any new semantics or CSS shadow boundary issues, and it puts markers in the ground for where to target our description lists in the future.